### PR TITLE
Adds loading by URI.

### LINKED ISSUE #1033: Load by URI
When search was added, loading by URI was removed. Per @michelleif it should still be available. Perhaps on the load tab?

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1033

### CODE REVIEW

File: __tests__/components/load/LoadByRDFForm.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 Stanford University see LICENSE for license
```
Comment (jcoyne) [inline __tests__/components/load/LoadByRDFForm.test.js]: 2019!

File: __tests__/components/load/LoadByURIForm.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 Stanford University see LICENSE for license
```
Comment (jcoyne) [inline __tests__/components/load/LoadByURIForm.test.js]: 2019!

File: __tests__/components/load/LoadResource.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2018 Stanford University see LICENSE for license
```
Comment (jcoyne) [inline __tests__/components/load/LoadResource.test.js]: 2019!

Comment (aaron-collier) [reply __tests__/components/load/LoadResource.test.js]: Doh! No more PR review before coffee for me. Missed that.

File: src/selectors/resourceSelectors.js [hunk 4]
```diff
@@ -57,7 +57,12 @@ export const resourceHasChangesSinceLastSave = (state) => {
   if (lastSaveChecksum === undefined) {
     return true
   }
-  const rdf = new GraphBuilder(state.selectorReducer).graph.toCanonical()
-  const resourceChecksum = generateMD5(rdf)
-  return lastSaveChecksum !== resourceChecksum
+  // This is temporary until the setting of lastSaveChecksum is fixed.
```
Comment (ndushay) [inline src/selectors/resourceSelectors.js]: is the problem referred to here ticketed?

File: src/components/load/LoadByRDFForm.jsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    rdfDatasetFromN3(resourceN3).then((dataset) => {
+      const builder = new ResourceStateBuilder(dataset, baseUri)
+      const resource = builder.state
+      dispatch(existingResourceCreator(resource))
+      props.history.push('/editor')
+    })
+  },
+})
+
+const mapStateToProps = state => ({
```
Comment (ndushay) [inline src/components/load/LoadByRDFForm.jsx]: OMG these methods are out of order!  Calamity!

File: src/selectors/resourceSelectors.js (hunk 4 above)
Comment (justinlittman) [reply src/selectors/resourceSelectors.js]: Being fixed in https://github.com/LD4P/sinopia_editor/pull/1111

**Pull request merged**